usaspending_api/broker/lookups.py

```diff
@@ -5,12 +5,14 @@
 # code.
 from collections import namedtuple
 
-LookupType = namedtuple('LookupType', ['id', 'name', 'desc'])
+LookupType = namedtuple("LookupType", ["id", "name", "desc"])
 
 EXTERNAL_DATA_TYPE = [
-    LookupType(1, 'fpds', 'fpds external data type'),
-    LookupType(2, 'fabs', 'fabs external data type'),
-    LookupType(3, 'exec_comp', 'exec external data type')
+    LookupType(1, "fpds", "FPDS transactions from Broker"),
+    LookupType(2, "fabs", "FABS transactions from Broker"),
+    LookupType(3, "exec_comp", "Executive Compensation from Broker"),
+    # "opposite" side of the broker data load, data from USAspending DB -> Elasticsearch
+    LookupType(100, "es_transactions", "Load elasticsearch with transactions from USAspending"),
```

Comment:
Why the arbitrary skip?

Reply:
The pre-existing types are for loading data into the USAspending database. This is for loading something else from the USAspending DB. It makes sense to leave room for more Broker -> USAspending types